Package.swift

```diff
@@ -124,7 +124,7 @@ if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
         .package(name: "swift-markdown", url: "https://github.com/apple/swift-markdown.git", .branch("release/5.7")),
         .package(name: "CLMDB", url: "https://github.com/apple/swift-lmdb.git", .branch("release/5.7")),
         .package(url: "https://github.com/apple/swift-argument-parser", .upToNextMinor(from: "1.0.1")),
-        .package(name: "SymbolKit", url: "https://github.com/apple/swift-docc-symbolkit", .branch("release/5.7")),
+        .package(name: "SymbolKit", url: "https://github.com/d-ronnqvist/swift-docc-symbolkit", .branch("check-doc-comment-module-info-5.7")),
```

Comment:
Please don't forget to undo this before merging. Thanks!

Reply:
I just need to wait for https://github.com/apple/swift-docc-symbolkit/pull/43 to be merged before undoing this change.

Reply:
I'm holding this until I can cherry-pick https://github.com/apple/swift-docc-symbolkit/pull/44 as well

Reply:
This is addressed in https://github.com/apple/swift-docc/pull/316/commits/094931d75128207ed6cf720b42f16c26b9106d13